### ISSUE TITLE
PWA Progressive Web Application

### ISSUE THREAD

**Issue by enrilor** [alextselegidis/easyappointments#1024]
Hi everybody, should be possible to add PWA functionality to easyappointments ?
Just for customer to saved on home screen and access easly .

**Comment by alextselegidis** [comment]
Hello! 

Adding PWA support is in general possible. 

Can you please elaborate a little bit more on what you need exactly?

<small>
<table>
<tr>
<td><img src="https://easyappointments.org/img/easyappointments-signature.png"></td>
<td>&nbsp;&nbsp;Alex Tselegidis, Easy!Appointments Creator
<br>&nbsp;&nbsp;<a href="https://alextselegidis.com">Need a customization? Contact me in person!</a></td>
</tr>
</table>
</small>

**Comment by enrilor** [comment]
Hi Alex,

The maily advantage is that the visitor will receive a popup to add the site to the home screen. In this case he will have an icon like a native app to use to book appointments.
Also notification with PWA could be a great feature, once the vistor installed the App could also receive notification from app (like booking confirmation or reminder), but it's more complicated feature I think. 
Thanks for great work that you are doing.

**Comment by alextselegidis** [comment]
Right, I also think that this and other advantages of PWA will come in handy for the future of the app so I will mark this as a future enhancement. 

<small>
<table>
<tr>
<td><img src="https://easyappointments.org/img/easyappointments-signature.png"></td>
<td>&nbsp;&nbsp;Alex Tselegidis, Easy!Appointments Creator
<br>&nbsp;&nbsp;<a href="https://alextselegidis.com">Need a customization? Contact me in person!</a></td>
</tr>
</table>
</small>
